dependencies.yaml

```diff
@@ -434,12 +436,10 @@ dependencies:
     common:
       - output_types: [conda, pyproject]
         packages:
-          - aiohttp
             # cudf will use fsspec but is protocol independent. cugraph tests
             # specifically require http for the test files it asks cudf to read.
           - fsspec[http]>=0.6.0
```

Comment:
Is this `fsspec[http]` pinning related to the original issue? That installs `aiohttp` and `requests` as dependencies. Perhaps the proper fix is actually to require `fsspec[http]`... but note that `fsspec[http]` is not valid syntax for a conda dependency (only pip understands the `[extras]` syntax).

I would propose the following:
- add `fsspec[http]` as a pyproject run dependency
- add `fsspec, aiohttp, requests` as conda run dependencies
- remove `fsspec[http]` as a test dependency

Reply:
Since `requests` and `aiohttp` are indeed run dependencies (see [here](https://github.com/rapidsai/cugraph/issues/2811#issuecomment-1682752849)) I agree with @bdice 's proposed changes.

I would also suggest changing the comment above stating that cugraph uses them for tests, since that's not entirely accurate.